packages/data/src/registry.js

```diff
@@ -166,10 +166,11 @@ export function createRegistry( storeConfigs = {} ) {
 		return registry;
 	}
 
-	Object.entries( {
-		'core/data': dataStore,
-		...storeConfigs,
-	} ).map( ( [ name, config ] ) => registry.registerStore( name, config ) );
+	registerGenericStore( 'core/data', createCoreDataStore( registry ) );
+
+	Object.entries( storeConfigs ).map(
```

Comment:
Nit: It's a good opportunity to stop using `Array#map` wrongly (vs. the more appropriate `Array#forEach`)

Reply:
ooo good catch there.